etc/pam.d/Makefile.am

```diff
@@ -2,20 +2,20 @@
 # and also cooperate to make a distribution for `make dist'
 
 pamd_files = \
+	chpasswd \
 	chfn \
 	chsh \
 	groupmems \
 	login \
+	newusers \
 	passwd
 
 pamd_acct_tools_files = \
 	chage \
-	chgpasswd \
```

Comment:
was that intentional? `chgpasswd != chpasswd` - the one added in pam_files does match the commit message to be `chpasswd`

Reply:
It looks like an accident.  <https://github.com/shadow-maint/shadow/issues/926#issuecomment-1941324761>

Reply:
It was an accident. Thank you for spotting this.

Reply:
Hi @loqs !

Would you mind testing the fix?

-  <https://github.com/shadow-maint/shadow/pull/949>

Thanks!

Have a lovely day!
Alex